omero/developers/Modules/Search.rst

```diff
@@ -39,7 +39,16 @@ Queries
 Search queries are very similar to Google searches. When search terms
 are entered without a prefix ("name:"), then the default field will be
 used which combines all available fields. Otherwise, a prefix can be
-added to restrict the search.
+added to restrict the search. The search terms will be by default parsed as if there was "OR" operator
```

Comment:
Maybe  "By default, the search terms will be parsed as if there was *an* "OR" operator...
 - Actually, what is non-default? Do you need the "by default" in this sentence?